### DISCUSSION TITLE
weblate 问题汇总

### DISCUSSION THREAD

**Discussion by exzhawk** [CFPAOrg/Minecraft-Mod-Language-Package#1046] [General]
**小图可以点开看大图**

现实情况
1. weblate上大项目连续性差
2. weblate上平均翻译质量低于github上pr平均翻译质量

已知原因/问题
1. (曾经)没有审核
2. diff history不好用(具体?)
3. weblate会影响语言文件的脉络(分版块, 删空行, 格式)
4. review麻烦(具体?)
5. 词汇表混乱
6. 词汇表不能区分mod, 只有所有mod的大表
    1. proposal1 每个mod分一个词汇表, 优先本mod <a href="https://user-images.githubusercontent.com/7038938/111368002-4ee6cc00-86d0-11eb-88b8-cf804d212826.png"><img src="https://user-images.githubusercontent.com/7038938/111368002-4ee6cc00-86d0-11eb-88b8-cf804d212826.png" height="200" ></a>

7. zen模式的界面也非常臃肿
    1. proposal1 翻译人员界面. 目前没有放词汇表(和其他单页模式下的一些功能)的地方 <a href="https://user-images.githubusercontent.com/7038938/111367496-c36d3b00-86cf-11eb-8bdc-a91557e89c91.png"><img src="https://user-images.githubusercontent.com/7038938/111367496-c36d3b00-86cf-11eb-8bdc-a91557e89c91.png" height="200"></a>

@TUsama @ChromaPIE @TartaricAcid

**Comment by TartaricAcid** [comment]
我有个想法：CurseForge 上热度高的项目也许可以不入 Weblate，只在 Weblate 上放小模组（得确定什么算小模组？以及哪些小模组应该入库？）

具体做法：
- CurseForge 上按下载量排名前 xxx 位的模组不入 Weblate，只接受 PR 提交翻译
- 下载量靠前的整合包里一般会包含很多下载量低的小模组，扣除上述提到的，剩下的入 Weblate
- 其他呼声比较高（怎么定义呼声高？）的模组入库（依据文本量大小和难易程度选择入 Weblate）

**Reply by n0099** [reply]
小模组：没什么人玩，作者跑路了的mod

**Comment by EnderAvaritia** [comment]
这个是直接对着文字干翻译吗？会脱离语境的吧？

**Reply by n0099** [reply]
面向vsc翻译
